packages/roosterjs-editor-api/lib/index.ts

```diff
@@ -18,6 +18,7 @@ export { default as editTable } from './table/editTable';
 export { default as formatTable } from './table/formatTable';
 export { default as removeLink } from './format/removeLink';
 export { default as replaceWithNode } from './format/replaceWithNode';
+export { default as rotateImage } from './format/rotateImage';

```

Comment:
Question: How do we decide if a feature should be a a plugin vs editor api? Specifically, I am wondering for image crop... I see ImageResize is a plugin.